packages/smooth_app/lib/cards/product_cards/smooth_product_card_thanks.dart

```diff
@@ -1,5 +1,4 @@
 import 'package:flutter/material.dart';
-
```

Comment:
These are from me running 'optimize imports' in Android Studio. Should I revert them or is it fine?

Reply:
dunno, if dart analyze doesn't complain then it should be ok to have

Reply:
Reverted those changes :)